src/current/_includes/releases/v22.2/v22.2.13.md

```diff
@@ -0,0 +1,138 @@
+## v22.2.13
+
+Release Date: August 9, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-13-security-updates">Security updates</h3>
+
+- The full set of TLS ciphers that was present in v22.1 have been included in the existing cipher suites list, which can be enabled with the `COCKROACH_TLS_ENABLE_OLD_CIPHER_SUITES` environment variable. [#105355][#105355]
+
+<h3 id="v22-2-13-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Fixed an issue where [changefeeds](../v22.2/change-data-capture-overview.html) emitting to a cloud storage sink with the [`compression` option](../v22.2/create-changefeed.html#compression-opt) may experience resource leakage (memory and go routines) when encountering transient errors. [#106856][#106856]
+- Fixed a rare [changefeed](../v22.2/change-data-capture-overview.html) issue that was triggered when the parent database or types were dropped. Instead of exiting with a descriptive error message, the changefeed would observe the opaque error `value type is not BYTES: UNKNOWN`. [#107937][#107937]
+
+<h3 id="v22-2-13-sql-language-changes">SQL language changes</h3>
+
+- Inbound foreign keys are now allowed on TTL tables. If the inbound foreign key has the [`ON DELETE CASCADE` option](../v22.2/foreign-key.html#foreign-key-actions), the [TTL job](../v22.2/row-level-ttl.html#view-running-ttl-jobs) will work as usual. If the inbound foreign key has the `ON DELETE RESTRICT` option, the TTL job will fail if a row references the TTL table. [#105342][#105342]
+- The [`crdb_internal.cluster_queries`](../v22.2/crdb-internal.html) and `crdb_internal.node_queries` tables will redact SQL queries if the user has the `VIEWACTIVITYREDACTED` [system privilege](../v22.2/security-reference/authorization.html#supported-privileges). If the user has the `VIEWACTIVITY` system privilege, these tables will not redact SQL queries. Users can now only view the `crdb_internal.cluster_queries` and `crdb_internal.node_queries` tables if they have any of the following privileges: `admin`, `VIEWACTIVITY`, or `VIEWACTIVITYREDACTED`. [#105551][#105551]
+- The [`kv.allocator.lease_rebalance_threshold`](../v22.2/cluster-settings.html) cluster setting can now be used to control the minimum fraction away from the mean a store's lease count is before it is considered for lease transfers. The default setting is `0.05`. [#106186][#106186]
+
+<h3 id="v22-2-13-db-console-changes">DB Console changes</h3>
+
+- Fixed a bug where the [**Job Details** page](../v22.2/ui-jobs-page.html#job-details) would flicker between the job details and a loading animation while a job is still executing. [#106156][#106156]
```

Comment:
Could move this to Bug Fixes too

Reply:
Moved this! Though again, I am actually unsure of what we should prioritize... DB Console or Bug Fixes for notes that fit under either.